Cesium.Core/Exceptions/WipException.cs

```diff
@@ -0,0 +1,17 @@
+namespace Cesium.Core.Exceptions;
```

Comment:
Can we have exceptions in Cesium namespace. Otherwise eventually we will have `using Cesium.Core.Exceptions;` everywhere. And I mostly do not see a reason why we would not need them.

Reply:
Agree on this point.